app/views/govuk_publishing_components/components/docs/intervention.yml

```diff
@@ -6,7 +6,10 @@ body: |
   that would be useful to them. This component would be used to add this personalised content and would
   indicate to the user that this is not normally part of the page, but has been added for them specifically.
 
-  The dismiss link points to the current URL with the "hide-intervention" query string parameter set to "true". The backend should check for this string in order to render the same page without the intervention.
+  The dismiss link works without Javascript by pointing to the current URL with the "hide-intervention" 
+  query string parameter set to "true". It's progressively enhanced and sets a cookie to remember that 
+  the user has dismissed the banner before. The cookie requires a "name" parameter, the value of which 
+  is stored in the cookie to distinguish which campaign banner the user has dismissed.
```

Comment:
I just want to check this... `name` isn't a required parameter for the component, what happens if one isn't passed?

Reply:
It's now required as discussed